documentation/commands/git-town.md

```diff
@@ -51,5 +52,15 @@ git town version
 > git town perennial-branches --remove qa
 > ```
 
+* *pull-branch-strategy*
+> Displays the pull branch strategy. (default: 'rebase')
+> This strategy is used when syncing a local feature branch with its upstream.
+>
+> Specify a strategy to set the pull branch strategy.
```

Comment:
I would remove either this line or line 61. They are pretty much equivalent in their role to help understand what this example does.